filedistribution/src/main/java/com/yahoo/vespa/filedistribution/FileReceiver.java

```diff
@@ -32,6 +38,92 @@
     private final FileReferenceDownloader downloader;
     private final File downloadDirectory;
     private final XXHash64 hasher = XXHashFactory.fastestInstance().hash64();
+    private final AtomicInteger nextSessionId = new AtomicInteger(1);
+    private final Map<Integer, Session> sessions = new HashMap();
```

Comment:
You need diamond syntax here to avoid compilation failure (we use -Werror)

Reply:
Fixed